### PR TITLE
Openshift deployment now supports different domains

### PR DESCRIPTION
**Pull request by liam-middlebrook**

### CODE REVIEW

**Comment by liam-middlebrook** [comment]
Don't merge yet, fixing bug.

**Comment by msoucy** [comment]
LGTM :shipit:

**Comment by ryansb** [comment]
This makes supplying a domain mandatory, right?

File: ofcourse/cli/__init__.py [hunk 1]
```diff
@@ -177,6 +178,8 @@ def openshift(verbose, app, user):
             cfg.write(token)
 
     api = get_api(token)
+    if not domain:
+        domain = api.domain_get()[1]
```
Comment (msoucy) [inline ofcourse/cli/__init__.py]: Gets default domain

**Comment by liam-middlebrook** [comment]
@ryansb nope, lines 181-182 make it so providing a domain is optional

**Comment by ryansb** [comment]
Legit.

**Pull request merged**